skynet/modules/ttt/rag/stores/faiss.py

```diff
@@ -0,0 +1,82 @@
+import asyncio
+import shutil
+import time
+
+from uuid import uuid4
+
+import faiss
+
+from langchain_community.docstore.in_memory import InMemoryDocstore
+from langchain_community.vectorstores import FAISS
+from langchain_community.vectorstores.utils import DistanceStrategy
+
+from skynet.env import use_s3, vector_store_path
+from skynet.logs import get_logger
+from skynet.modules.ttt.rag.vector_store import SkynetVectorStore
+
+log = get_logger(__name__)
+
+
+class FAISSVectorStore(SkynetVectorStore):
+    def __init__(self):
+        super().__init__()
+
+        if use_s3:
+            from skynet.modules.ttt.rag.stores.s3 import RagS3
+
+            log.info('Using S3 for vector store')
```

Comment:
Can you reword this, since S3 is not the actual vector store but rather where we store the vestor store? (yeah that sounded weird in my head too)

Reply:
rewording to `Using S3 for vector store persistence`